modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/image/ImageDDMFormFieldTemplateContextContributor.java

```diff
@@ -93,6 +94,10 @@
 			DDMFormFieldTypeUtil.getPropertyValue(
 				ddmFormField, ddmFormFieldRenderingContext.getLocale(),
 				"predefinedValue")
+		).put(
+			"requiredDescription",
+			GetterUtil.getBoolean(
+				ddmFormField.getProperty("requiredDescription"))
```

Comment:
Hey, @IstvanD. In regards to the bug, wouldn't passing the default boolean as `true` here resolve the issue?

Reply:
Hi @rodrigopaulino thanks for the hint, but unfortunately, it didn't resolve it. The required description switch is still off.

Reply:
[Here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/java/com/liferay/dynamic/data/mapping/form/builder/internal/context/DDMFormBuilderContextFactoryHelper.java#L267) the value will be null because ddmFormField won't have it `requiredDescription` property set.

This will later be converted to **false**.

Reply:
Hi @rodrigopaulino what would be the proper way to solve this issue? Should perhaps `ImageDDMFormFieldTypeSettings` be used to get the `predefinedValue` for `requiredDescription`? Thank you in advance.

Reply:
Hey, @IstvanD. I believe that just setting the `requiredDescription` to `true` wherever it doesn't exists might work.

Reply:
Adding something like this works, although its quite ugly
https://github.com/IstvanD/liferay-portal/commit/250d20d7c2608ee9bd690f4e166b9c99c0957556

I would prefer to add the logic in [doCreateDDMFormFieldValue()](https://github.com/IstvanD/liferay-portal/blob/250d20d7c2608ee9bd690f4e166b9c99c0957556/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/java/com/liferay/dynamic/data/mapping/form/builder/internal/context/DDMFormBuilderContextFactoryHelper.java#L293) but its arguments don't contain data about the type of the `ddmFormField`.